File: Data Structures & Algorithms/is-anagram/submission-4.java
class Solution {
    public boolean isAnagram(String s, String t) {

        int[] res = new int[26];

        if(s.length() != t.length()) return false;

        for(int i=0; i<s.length(); i++){
            char cs = s.charAt(i);
            char ct = t.charAt(i);

            res[cs - 'a']++;
            res[ct - 'a']--;
        }

        for(int i=0; i<res.length; i++){
            if(res[i] == -1) return false;
        }
        return true;
    }
}

File: Data Structures & Algorithms/baseball-game/submission-4.py
class Solution:
    def calPoints(self, operations: List[str]) -> int:
        stack = []

        for op in operations:

            if op == "+":
                x = int(stack.pop())
                y = int(stack.pop())

                total = x + y
                stack.append(y)
                stack.append(x)
                stack.append(total)
            elif op == "C":
                stack.pop()
            elif op == "D":
                x = int(stack[-1])
                x *= 2
                stack.append(x)
            else:
                stack.append(int(op))
            print(op)
            print(stack)
        res = 0
        while stack:
            res += int(stack.pop())
        
        return res
